src/quo2/reanimated.cljs

```diff
@@ -0,0 +1,69 @@
+(ns quo2.reanimated
+  (:require ["react-native" :as rn]
+            [reagent.core :as reagent]
+            [clojure.string :as string]
+            ["react-native-reanimated" :default reanimated
+             :refer (useSharedValue useAnimatedStyle withTiming withDelay Easing Keyframe)]))
+
+;; Animated Components
+(def create-animated-component (comp reagent/adapt-react-class (.-createAnimatedComponent reanimated)))
+
+(def view (reagent/adapt-react-class (.-View reanimated)))
+(def image (reagent/adapt-react-class (.-Image reanimated)))
+(def touchable-opacity (create-animated-component (.-TouchableOpacity ^js rn)))
+
+;; Hooks 
+(def use-shared-value useSharedValue)
+(def use-animated-style useAnimatedStyle)
+
+;; Animations
+(def with-timing withTiming)
+(def with-delay withDelay)
+(def key-frame Keyframe)
+
+;; Easings
+(def bezier (.-bezier ^js Easing))
+
+(def easings {:linear  (bezier 0 0 1 1)
+              :easing1 (bezier 0.25 0.1 0.25 1) ;; TODO(parvesh) - rename easing functions, (design team input)
+              :easing2 (bezier 0 0.3 0.6 0.9)
+              :easing3 (bezier 0.3 0.3 0.3 0.9)})
+
+;; Helper functions
+(defn get-shared-value [anim]
+  (.-value anim))
+
+(defn set-shared-value [anim val]
+  (set! (.-value anim) val))
+
+(defn kebab-case->camelCase [k]
+  (let [words (string/split (name k) #"-")]
+    (->> (map string/capitalize (rest words))
+         (apply str (first words))
+         keyword)))
+
+(defn map-keys [f m]
+  (->> (map (fn [[k v]] [(f k) v]) m)
+       (into {})))
+
+;; Worklets
+(def worklet-factory (js/require "../js/worklet_factory.js"))
+
+;;;; Component Animations
+
+;; kebab-case styles are not working for worklets
+;; so first convert kebab case styles into camel case styles
+(defn apply-animations-to-style [animations style]
+  (let [animations (map-keys kebab-case->camelCase animations)
```

Comment:
Hi, I think kebab case only works with clojure due to reagent, but as js directive inside js file which is directly processed by metro instead of shadow-cljs (due to js/require) only camel case works there